ios/CameraView+TakePhoto.swift

```diff
@@ -78,6 +78,10 @@ extension CameraView {
       if #available(iOS 12.0, *) {
         photoSettings.isPortraitEffectsMatteDeliveryEnabled = photoOutput.isPortraitEffectsMatteDeliveryEnabled
       }
+        
+      if self.enableDepthData?.boolValue == true {
+        photoSettings.embedsDepthDataInPhoto = true;
```

Comment:
Are you sure that this works? From reading the documentation of that prop, it says that it is enabled by default..

Also I'd prefer the direct assignment:

```swift
photoSettings.embedsDepthDataInPhoto = photoSettings.isDepthDataDeliveryEnabled
```

Reply:
It should be yeah - I'll remove this and set it up that it just takes the available `depthData` CVPixelBuffer of the captured image and writes this to an additional file for now. I think this will help keep it more modular and easier to access than having to try and decode the depth data out of the saved JPEG directly or through metadata which seems really odd and convoluted in the first place for Apple to embed it like that